From 697d434c0b8d42bd7453cadfaa57da44713790a2 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Wed, 3 Oct 2018 23:30:49 -0700
Subject: [PATCH 07/10] DO-NOT-UPSTREAM: iavf: Disable -Wignored-attributes

Should be easy to tackle when Miguel Ojeda's compiler attributes
patchset hits -next.

Link: https://github.com/ClangBuiltLinux/linux/issues/111
Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 drivers/net/ethernet/intel/iavf/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/intel/iavf/Makefile b/drivers/net/ethernet/intel/iavf/Makefile
index 9cbb5743ed12..0f3456cf5e30 100644
--- a/drivers/net/ethernet/intel/iavf/Makefile
+++ b/drivers/net/ethernet/intel/iavf/Makefile
@@ -7,7 +7,7 @@
 #
 
 ccflags-y += -I$(src)
-subdir-ccflags-y += -I$(src)
+subdir-ccflags-y := -I$(src) $(call cc-disable-warning, ignored-attributes)
 
 obj-$(CONFIG_IAVF) += iavf.o
 
-- 
2.19.1

